service/db/src/main/java/gov/va/vro/service/db/ClaimMetricsServiceImpl.java

```diff
@@ -126,10 +126,23 @@ private Page<ExamOrderEntity> findAllExamOrderInfoPage(ExamOrderInfoQueryParams
   }
 
   @Override
-  public ExamOrdersInfo findAllExamOrderInfo(ExamOrderInfoQueryParams params) {
+  public ExamOrdersInfo findExamOrderInfo(ExamOrderInfoQueryParams params) {
     Page<ExamOrderEntity> examOrders = findAllExamOrderInfoPage(params);
-    List<ExamOrderInfoResponse> examOrdersInfo =
+    List<ExamOrderInfoResponse> allExamOrdersInfo =
         examOrderInfoResponseMapper.toExamOrderInfoResponses(examOrders);
-    return new ExamOrdersInfo(examOrdersInfo, examOrders.getTotalElements());
+    List<ExamOrderInfoResponse> notOrderedExamOrdersInfo = new ArrayList<>();
+    List<ExamOrderInfoResponse> response;
```

Comment:
Use JPA not manual filtering. Paging will not work right the way you are doing either. findByOrderAtIsNull should work I believe.